test/integration/toolchains/microsoft/test_msbuilddeps.py

```diff
@@ -122,3 +123,52 @@ def package_info(self):
         assert "conan_liba" not in libb
         libb = c.load("app/conan_libb_mycomp_vars_release_x64.props")
         assert "conan_liba" not in libb
+
+
+def test_msbuilddeps_relocatable():
```

Comment:
```suggestion
@pytest.mark.skipif(platform.system() != "Windows", reason="Requires Windows")
def test_msbuilddeps_relocatable():
```

Reply:
I wrote a bit different comment, since the test does not require Windows in theory.
I guess it just does not work properly with POSIX paths and breaks their leading slash. It does not matter of course.